cachito/web/models.py

```diff
@@ -162,6 +175,18 @@ def number_of_deps(self):
         return db.session.query(sqlalchemy.func.count(req_dep_cols.dependency_id)).filter(
             req_dep_cols.request_id == self.id).scalar()
 
+    @property
+    def dependency_overrides_count(self):
+        """
+        Get the total number of dependency overrides for a request.
+
+        :return: the number of dependency overrides
+        :rtype: int
+        """
+        req_dep_cols = request_dependency_override_table.c
+        return db.session.query(sqlalchemy.func.count(req_dep_cols.dependency_id)).filter(
+            req_dep_cols.request_id == self.id).scalar()
```

Comment:
Didn't know about [scalar](https://docs.sqlalchemy.org/en/13/orm/query.html#sqlalchemy.orm.query.Query.scalar). We should use it more often.